getting-started/data-setup.rst

```diff
@@ -46,149 +44,91 @@ It may simply be that you're working in a brand new shell.
-=============================================
-
-Next, populate the repository with data from `testdata_ci_hsc`_.
-The Pipelines' :command:`ingestImages.py` command (called a **command-line task**) links raw images into a Butler repository, allowing the mapper to organize the data.
-Run:
-
-.. code-block:: bash
-
-   ingestImages.py DATA $TESTDATA_CI_HSC_DIR/raw/*.fits --mode=link
-
-.. tip::
-
-   Notice that the first argument to most command-line tasks is the Butler repository.
-   In this case it's the :file:`DATA` directory.
-
-.. tip::
-
-   You can learn about the arguments for command-line tasks with the ``-h`` flag.
-   For example:
-
-   .. code-block:: bash
+The `gen3_rc2_subset`_ git repository has a Butler repository contained within it.
+To construct a Butler that can manage data in that repository, from a python prompt say:
 
-      ingestImages.py -h
+.. code-block:: python
 
-Install transmission curves
-===========================
+   from lsst.daf.butler import Butler
+   import os
+   butler = Butler(os.environ['GEN3_DC2_SUBSET_DIR'] + '/SMALL_HSC')
 
-Run this command to install transmission curves corresponding to the raw data:
+Now you can explore the repository using the registry atribute of the Butler you created.  E.g.:
 
-.. code-block:: bash
-
-   installTransmissionCurves.py DATA
-
-Transmission calibrations, like this, are currently a special feature for HSC data `implemented in the obs_subaru package <https://github.com/lsst/obs_subaru/tree/master/hsc/transmission>`_.
+.. code-block:: python
 
-Ingesting calibrations into the Butler repository
-=================================================
+   registry = butler.registry
+   for col in registry.queryCollections():
+       print(col)
+   for ref in registry.queryDatasets('raw', collections='', instrument='HSC'):
+       print(ref.full)
 
-Next, add calibration images (such as dark, flat, and bias frames) associated with the raw data:
-
-.. code-block:: bash
+Notes on processing
+===================
 
-   ln -s $TESTDATA_CI_HSC_DIR/CALIB/ DATA/CALIB
+The intention of this set of introductory recipes is to give you a realistic sense of how data is processed using the LSST Science Pipelines.
+That includes taking raw images all the way through to coaddition and forced photometry.
+Though the starting repository is small, a significant amount of processing needs to be done to produce all the datasets needed for downstream processing.
+This means that some steps can be quite time consuming and you should be prepared to wait or perhaps run things overnight if you intend to follow these examples line by line.
 
-.. note::
+The most time consuming steps are:
 
-   In general, you can use the :command:`ingestCalibs.py` command-line task to ingest calibrations into a Butler repository.
-   For this tutorial, we've taken a shortcut by manually symlinking pre-structured calibrations from the `testdata_ci_hsc`_ package.
+- Single frame processing: 11 hours
```

Comment:
Presumably for initial demonstration purposes you could have them process a single frame by restricting the query?

Reply:
I'm going to leave it as is and if we need to add a tip box about how to get started if you don't want to go on to other tutorials, we can